API/Services/ForecastDataRetrievalService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Globalization;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+using Backend.API.MetAPI;
+
+namespace Backend.API.Services
+{
+    public interface IForecastDataRetrievalService
+    {
+        //HttpClient HttpClient { get; init; }
```

Comment:
Commented code generally means that it's not going to be used, and should thus not be present in the delivered code.